component/src/main/java/org/wso2/extension/siddhi/gpl/execution/pmml/PmmlModelProcessor.java

```diff
@@ -92,12 +83,12 @@
                 )
         },
         returnAttributes = {
-            @ReturnAttribute(
-                    name = "output",
-                    description = "All the processed outputs defined in the query. The number of outputs can " +
-                            "vary depending on the query definition.",
-                    type = {DataType.OBJECT}
-            )
+                @ReturnAttribute(
+                        name = "output",
+                        description = "All the processed outputs defined in the query. The number of outputs can " +
+                                "vary depending on the query definition.",
+                        type = {DataType.OBJECT}
```

Comment:
Is it object ?

Reply:
It can be any type(String, double, float...) specified within the model